{"uuid":"02022c71-504b-411c-9100-4e63f0700ee7","historyId":"36030a09f9b1923da241513131a6c206","fullName":"tests.ErrorValidationTest.testInvalidLoginLandingPage","labels":[{"name":"package","value":"tests.ErrorValidationTest"},{"name":"testClass","value":"tests.ErrorValidationTest"},{"name":"testMethod","value":"testInvalidLoginLandingPage"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"tests.ErrorValidationTest"},{"name":"host","value":"Qcentris-396"},{"name":"thread","value":"24388@Qcentris-396.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Validates error message for invalid login attempts on the landing page.","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.AssertionError","trace":"java.lang.AssertionError\r\n\tat org.junit.Assert.fail(Assert.java:87)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat org.junit.Assert.assertTrue(Assert.java:53)\r\n\tat tests.ErrorValidationTest.testInvalidLoginLandingPage(ErrorValidationTest.java:59)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\n"},"stage":"finished","description":"This test performs an invalid login and validates the error message displayed on the landing page.","steps":[],"attachments":[{"name":"Test Body","source":"84b8161e-8988-49ae-bf04-c8f5afdff8a6-attachment.txt","type":"text/plain"},{"name":"Invalid Login Error Screenshot","source":"388cd9ae-4852-4f42-8604-cde53f7a4308-attachment"},{"name":"Error Details","source":"e1b32ac3-83f5-4177-9012-978df64e2cdf-attachment.txt","type":"text/plain"}],"parameters":[],"start":1736069160438,"stop":1736069160772}